internal/command/operator.go

```diff
@@ -73,26 +72,23 @@ func operatorDeploy() *cobra.Command {
 
 				http := client.New(ctx, apiURL)
 
-				serviceAccounts, err := subscription.CreateGoogleServiceAccount(
-					ctx,
-					http,
-					cnf.Organization,
-					fmt.Sprintf("%s-jsctl-auto", cnf.Organization),
-				)
-				if err != nil || len(serviceAccounts) < 1 {
-					return fmt.Errorf("failed to create registry credentials: %w", err)
+				// TODO: this would ideally come from the config in ctx
+				configDir, err := os.UserConfigDir()
+				if err != nil {
+					return err
 				}
 
-				registryCredentialsBytes, err := base64.StdEncoding.DecodeString(serviceAccounts[0].Key.PrivateData)
+				registryCredentialsBytes, err := registry.FetchOrLoadJetstackSecureEnterpriseRegistryCredentials(ctx, http, configDir)
 				if err != nil {
-					return fmt.Errorf("failed to decode registry credentials: %w", err)
+					return fmt.Errorf("failed to fetch or load registry credentials: %s", err)
```

Comment:
Creds fetching is not enabled for my org:
```
irbe@jsctl$ go run main.go operator deploy
failed to fetch or load registry credentials: failed to create registry credentials: operation not allowed: feature disabled for org (401)
exit status 1
```

Should we instead of erroring here, print something like 'registry credentials not provided and unable to fetch credentials for this org. Make sure operator image is available in/from your cluster' or similar?

Reply:
Users may have provided path to an internal registry that doesn't need creds for example, so wouldn't make sure to error out just because creds can't be fetched from UI at this point I guess

Reply:
I have changed this error so that if they try and fetch the creds with the new flag they get:

> failed to create registry credentials, current organization "jetstack-charlie" does not have permissions to access the Jetstack Secure Enterprise registry. Please contact support if this is unexpected.